replication.go

```diff
@@ -2089,7 +2026,7 @@ func (cm *ContentManager) makeDealsForContent(ctx context.Context, content util.
 	))
 	defer span.End()
 
-	if content.Size < (256 << 10) {
+	if content.Size < constants.IndividualDealThreshold {
```

Comment:
update the error message with the fact that the content size is below the deal sie threshold